asapdiscovery/simulation/scripts/MProP0009_Sub.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#BSUB -J run_single_simulation
+#BSUB -oo log_files/MProP0009_Sub.out
+#BSUB -e log_files/MProP0009_Sub.stderr
+#BSUB -n 1
+#BSUB -R rusage[mem=4]
+#BSUB -q gpuqueue
+#BSUB -W 12:00
+mkdir -p /data/chodera/lemonsk/asap-datasets/prepped_mpro_P0009/
```

Comment:
This directory and the directory in line 16 should be the same, right? Since that's what you're actually writing out to. The other thing, which is potentially better, would be to delete this line and just have in your script a little line that creates the directory passed to `-o` if it doesn't exist.

Reply:
I see, as another alternative can I just go into my directory and just make this new directory altogether?

Reply:
sure! that can get annoying if you want to do that many times for many different things but it will work fine here!